### PR TITLE
fix(widget-space): Button style collision

### PR DESCRIPTION
**Pull request by bzang**
Ended up only fixing the specific issue. Main issue is that we're passing classes down into components which breaks css modules.

### CODE REVIEW

File: packages/node_modules/@ciscospark/widget-message-meet/src/components/activity-menu/index.js [hunk 1]
```diff
@@ -26,10 +26,12 @@ export default function MessageMeetActivityMenu({
     }
   ];
   return (
-    <ActivityMenu
-      buttons={buttons}
-      onExit={onExit}
-    />
+    <div className={styles.activityMenuContainer}>
```
Comment (adamweeks) [inline packages/node_modules/@ciscospark/widget-message-meet/src/components/activity-menu/index.js]: Do we need a user configurable style name here?

File: packages/node_modules/@ciscospark/widget-message-meet/src/components/activity-menu/styles.css [hunk 2]
```diff
@@ -1,7 +1,11 @@
...
+.activity-menu-container {
+  height: 100%;
+}
+
+.activity-menu-container .message-button {
   background-color: #07c1e4;
 }
 
-.meet-button {
+.activity-menu-container .meet-button {
```
Comment (adamweeks) [inline packages/node_modules/@ciscospark/widget-message-meet/src/components/activity-menu/styles.css]: Good call

File: scripts/webpack/webpack.base.babel.js [hunk 3]
```diff
@@ -18,7 +18,12 @@ export default (options) => {
     ]),
     new ExtractTextPlugin({filename: `[name].css`, disable: false, allChunks: true}),
     // Adds use strict to prevent catch global namespace issues outside of chunks.
-    new webpack.BannerPlugin(`react-ciscospark v${packageJson.version}`)
+    new webpack.BannerPlugin(`react-ciscospark v${packageJson.version}`),
+    new webpack.LoaderOptionsPlugin({
```
Comment (adamweeks) [inline scripts/webpack/webpack.base.babel.js]: Should we put a note that this is just a temporary change until the plugins support context config?

Comment (bzang) [inline scripts/webpack/webpack.base.babel.js]: I couldn't find any reference for when they'd add it. It seems to just be the accepted "fix".

File: packages/node_modules/@ciscospark/widget-message-meet/src/components/activity-menu/index.js (hunk 1 above)
Comment (bzang) [inline packages/node_modules/@ciscospark/widget-message-meet/src/components/activity-menu/index.js]: There's a parent container that already has a user configurable name... this is just so we have some context in the component to grab onto with our scoped styles.

File: scripts/webpack/webpack.base.babel.js (hunk 3 above)
Comment (adamweeks) [inline scripts/webpack/webpack.base.babel.js]: Oh lord. https://webpack.js.org/plugins/loader-options-plugin/ says
>This plugin will be removed in the future as it only exists for migration.

File: packages/node_modules/@ciscospark/widget-message-meet/src/components/activity-menu/index.js (hunk 1 above)
Comment (adamweeks) [inline packages/node_modules/@ciscospark/widget-message-meet/src/components/activity-menu/index.js]: Ok, I'm sure we'll need to go through at some point and make sure everything is user configurable, but that is WAY down the priority list.

File: scripts/webpack/webpack.base.babel.js (hunk 3 above)
Comment (bzang) [inline scripts/webpack/webpack.base.babel.js]: Oh snap! Well guess we probably shouldn't keep that in there then.

**Pull request merged**